src/action.js

```diff
@@ -11,9 +11,9 @@ async function run() {
   try {
     const { payload, eventName } = github.context;
 
-    if (eventName !== "pull_request") {
+    if (eventName !== "pull_request_target") {
```

Comment:
I don't think we should be doing this change. I think the change we do in source code should support both event names but we will recommend to use pull_request_target in our examples. There is also a problem of backwards compatibility, if we do the change in this way we are introducing a breaking change, which we should avoid

Reply:
After checking documentation [here](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target) it says: 

pull_request_target: 
"Runs your workflow when activity on a pull request in the workflow's repository occurs. For example, if no activity types are specified, the workflow runs when a pull request is opened or reopened or when the head branch of the pull request is updated." -- is exactly what we need.

If we support both events one can fail with "Resource not accessible by integration" error

Reply:
Yes, let's accept that this may happen as a compromise in order to avoid introducing a breaking change.

Reply:
But the opposite would happen too, If it is allowed to create comment will be two comments. One comment for each event.

Maybe to avoid introducing a breaking change we can just change the `log` message (as you proposed) as we know the limitation and it only happen when adding new comment from an external PR.

So we would not need to update all repositories that uses this action and would not introducing a new break change. 

What do you think?

Reply:
> But the opposite would happen too, If it is allowed to create comment will be two comments. One comment for each event.

This is not what will happen, it will not create two comments. Think about it and you can easily understand why it won't.